azkaban-common/src/main/java/azkaban/utils/StdOutErrRedirect.java

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright 2016 LinkedIn Corp.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package azkaban.utils;
+
+import java.io.PrintStream;
+
+import org.apache.log4j.Logger;
+import org.apache.log4j.Level;
+
+public class StdOutErrRedirect {
+  private static final Logger logger = Logger.getLogger(StdOutErrRedirect.class);
+
+  public static void redirectOutAndErrToLog() {
+    System.setOut(createStream(System.out, Level.INFO));
```

Comment:
In the solo server case, there will be 2 PrintStream objects created. What do you think about maintaining a single instance of each in the class itself?

Do you see any advantages of creating a fresh stream every time a redirect is called ?